force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,44 @@
+public class AccountTriggerHandler{
+    public static void setTypeProspect(List<Account> accNewList){
+        List<Account> accsForUpdate = new List<Account>();
+        for (Account acc : accNewList) {
+            if (acc.Type == null) {
+                acc.Type ='Prospect'; 
+                accsForUpdate.add(acc);
+            } 
+        }
+    }
+    public static void addressCopy(List<Account> accNewList){
+        List<Account> accsToUpdate = new List<Account>();
+        for (Account acc : accNewList) {
+            if (acc.ShippingStreet != null) {
+                acc.BillingStreet = acc.ShippingStreet;
+                acc.BillingCity = acc.ShippingCity;
+                acc.BillingState = acc.ShippingState;
+                acc.BillingPostalCode = acc.ShippingPostalCode;
+                acc.BillingCountry = acc.ShippingCountry;
+                accsToUpdate.add(acc);
+            }
```

Comment:
Seems like this only checks to see if the street is empty, Try to find out if the other fields are empty also. I know it looks bad to have a bunch of if statements but that's the requirement for this exercise.